### ISSUE TITLE
The doc website has poor contrast

### ISSUE THREAD

**Issue by remyabel** [talos-systems/talos#410]
The grey text on white background is already hard enough to read as it is, but it is even worse for those with accessibility needs. The WCAG [stipulates](https://dequeuniversity.com/rules/axe/3.1/color-contrast?application=lighthouse) that there should be a color contrast of at least 4.5:1 for small text or 3:1 for large text. Using the calculator, I think that something like `#4C5A5C` for the text color (or darker) would work.

**Comment by andrewrynhard** [comment]
@remyabel You are seeing my limitations in web development :D. In fact, I would like to find someone more knowledgable on this stuff than me. I threw up my best attempt :D. Is fixing this something that interests you? I would gladly accept a PR. The docs can be generated with `hugo`, so the development of it should be approachable.

**Comment by andrewrynhard** [comment]
@remyabel Closing this since we have moved the docs to a dedicated repo and are working on a new website. Thanks for the suggestion!
